src/derivatives/atlas.md

```diff
@@ -0,0 +1,961 @@
+            "tpl-MNI152NLin2009cAsym_atlas-MIAL67ThalamicNuclei_res-1_probseg.nii.gz": "",
+         },
+      },
+      "tpl-MNI152NLin6Asym": {
+         "anat": {
+            "tpl-MNI152NLin6Asym_atlas-MIAL67ThalamicNuclei_res-1_dseg.nii.gz": "",
+            "tpl-MNI152NLin6Asym_atlas-MIAL67ThalamicNuclei_res-1_probseg.nii.gz": "",
+         },
+      },
+   }
+})
+}}
+
+The next subsection describes this latter use-case in further depth.
+
+## Filenames of derivatives with atlases in their provenance
+
+Like for the [`space-<label>` entity](../glossary.md#space-entities),
+outputs derived from atlases MUST employ
+[`atlas-<label>`](../glossary.md#atlas-entities),
+[`seg-<label>`](../glossary.md#segmentation-entities), and
+[`scale-<label>`](../glossary.md#scale-entities) when necessary:
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+   "bold-pipeline": {
+      "atlas-Schaefer2018_dseg.json": "",
+      "atlas-Schaefer2018_seg-7n_scale-100_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-7n_scale-200_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-7n_scale-300_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-17n_scale-100_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-17n_scale-200_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-17n_scale-300_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-kong17n_scale-100_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-kong17n_scale-200_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-kong17n_scale-300_dseg.tsv": "",
+      "sub-01": {
+         "anat": {
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-7n_scale-100_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-7n_scale-200_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-7n_scale-300_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-17n_scale-100_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-17n_scale-200_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-17n_scale-300_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-kong17n_scale-100_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-kong17n_scale-200_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-kong17n_scale-300_den-164k_dseg.label.gii": "",
+            "...": "",
+            "sub-01_hemi-R_atlas-Schaefer2018_seg-kong17n_scale-300_den-164k_dseg.label.gii": "",
+         },
+         "bold": {
+            "sub-01_task-rest_hemi-L_den-164k_bold.func.gii": "",
+            "sub-01_task-rest_hemi-R_den-164k_bold.func.gii": "",
+         }
+      },
+   }
+})
+}}
+
+## Tabular data
+
+The `[probseg|dseg|mask|channels].tsv` file indexes and labels each node/parcel/region within the atlas.
```

Comment:
Let's remove channels.tsv from this context. It may behave similarly, but it doesn't fit the "atlas" concept being defined here.